GiniVision/Classes/Networking/DocumentServiceProtocol.swift

```diff
@@ -179,13 +179,13 @@ extension DocumentServiceProtocol {
                 if let error = task?.error {
                     let id = self.compositeDocument?.documentId ?? ""
                     let message = "Error sending feedback for document with id: \(id) error: \(error)"
-                    Logger.debug(message: message, event: .error)
+                    Logger.log(message: message, event: .error)
                     
                     return nil
                 }
                 
-                Logger.debug(message: "Feedback sent with \(updatedExtractions.count) extractions",
-                    event: .custom(emoji:"🚀"))
+                Logger.log(message: "Feedback sent with \(updatedExtractions.count) extractions",
+                    event: .custom("🚀"))
```

Comment:
I'm still a little skeptical about this. It's a little shorter, but still a bit verbose. It's way longer than just having the emoji inline in the string. And actually, do you really need custom ones?

Reply:
The idea is to represent the event in a graphical way, but since there is not way to check if it is an emoji or just a string maybe it makes sense to pass just a `String`.